README.md

```diff
@@ -74,23 +120,55 @@ if __name__ == "__main__":
         print("error occurred.", exc)
 ```
 
-#### Run File Uploader
+**1. Create a test file containing data:**
+
+You can do this with `dd` on Linux or macOS systems:
+
 ```sh
-$ python file_uploader.py
-'/home/user/Photos/asiaphotos.zip' is successfully uploaded as object 'asiaphotos-2015.zip' to bucket 'asiatrip'.
+dd if=/dev/urandom of=/tmp/minio-python-sdk-test-file.bin bs=2048 count=10
```

Comment:
These are all over complicating to run the example. My suggesting is to leave the prerequisite steps to be outside of this README.

Reply:
Dropped this for a different approach.